dnf-behave-tests/features/plugins-core/needs-restarting-conf-files.feature

```diff
@@ -0,0 +1,122 @@
+Feature: Add package to needs-restarting using config files
+
+Background:
+    Given I enable plugin "needs_restarting"
+      And I use repository "dnf-ci-fedora"
+      And I move the clock backward to "before boot-up"
+      And I execute dnf with args "install lame kernel basesystem glibc wget abcde"
+      And I move the clock forward to "the present"
+      And I use repository "dnf-ci-fedora-updates"
+    Given I create directory "/etc/dnf/plugins/needs-restarting.d"
+      And I create and substitute file "/etc/dnf/plugins/needs-restarting.d/wget.conf" with
+      """
+      wget
+      """
+      Given I do not set config file
+
+    @bz1810123
+    Scenario: restarting needed after updating package in conf files
```

Comment:
The alignment in the whole file looks quite weird. We don't really have a consistent way to align in feature files, sadly... but this is quite a bit different. I prefer something like:
```
Scenario: test
Given something
  And something else
 When X
 Then foo is
      """
      hello
      """
```